src/types/Barn.ts

```diff
@@ -11,4 +11,5 @@ export interface BarnFraAPI {
     mellomnavn?: string;
     etternavn?: string;
     fødselsdato: string;
+    uuid: string;
```

Comment:
Hva vil en `uuid` på et barn si?

Reply:
`uuid` på barn brukes kun i frontend for å sørge for at vi får plukket opp riktig barn når vi skal poste søknaden til backend, fordi pt sender vi ikke ned noe fødselsnummer eller annet som unikt identifiserer et barn til klienten.

Reply:
Skjønner! Takk for god forklaring :)